src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -258,7 +259,11 @@
     @Override
     protected int doWork() {
         initialize();
-        basecallsConverter.processTilesAndWritePerSampleOutputs(barcodeSamWriterMap.keySet());
+        try {
+            basecallsConverter.processTilesAndWritePerSampleOutputs(barcodeSamWriterMap.keySet());
+        } catch (IOException e) {
+            return 1;
```

Comment:
Maybe rethrow the exception as a PicardException with more information about what might have gone wrong?